pkg/database/token.go

```diff
@@ -77,8 +85,8 @@ func (s *Subject) SymptomInterval() uint32 {
 
 func ParseSubject(sub string) (*Subject, error) {
 	parts := strings.Split(sub, ".")
-	if length := len(parts); length < 1 || length > 2 {
-		return nil, fmt.Errorf("subject must contain 2 parts, got: %v", length)
+	if length := len(parts); length < 2 || length > 3 {
```

Comment:
Won't think break existing JWT tokens that have the old subject?

Reply:
the old if statement was wrong - it should have always been 2 parts before.

Why I didn't write `length != 2` the first time, we'll never know.

Reply:
Not that I doubt your code, but this feels like a test.

Reply:
^ what Jeremy said

Reply:
funny enough - the 2 segment subject was the only thing under test before.
test coverage has been boosted to cover all scenarios.